tests/stub/iteration/scripts/v4x0/tx_pull_1_nested.script

```diff
@@ -17,14 +17,14 @@ S: SUCCESS {"fields": ["x"], "qid": 2}
    SUCCESS {"has_more": true}
 C: PULL {"n": 1, "qid": 2}
 S: RECORD ["2_2"]
-   SUCCESS {"has_more": false}
+   SUCCESS {}
```

Comment:
Why remove the `"has_more": false`?

Reply:
This is what the server does. I feel like if we want to test that the driver can handle an explicit `"has_more": false` we should probably have a separate test for it.